File: app/src/main/java/n7/ad2/tournaments/db/TournamentGame.java
package n7.ad2.tournaments.db;

import androidx.room.Entity;
import androidx.room.PrimaryKey;
import androidx.annotation.NonNull;

@Entity
public class TournamentGame {

    @PrimaryKey
    @NonNull
    public String url;

    public String team1Name = ".";
    public String team2Name = ".";
    public String team1Logo = ".";
    public String team2Logo = ".";
    public String teamScore = ".";
    public long teamTimeRemains = 0;
    public long teamTime = 0;

}

File: feature/tournaments/src/main/kotlin/n7/ad2/tournaments/internal/db/TournamentGame.kt
package n7.ad2.tournaments.internal.db

import androidx.annotation.NonNull
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
data class TournamentGame(
    @PrimaryKey
    @NonNull
    var url: String = "",

    var team1Name: String = ".",
    var team2Name: String = ".",
    var team1Logo: String = ".",
    var team2Logo: String = ".",
    var teamScore: String = ".",
    var teamTimeRemains: Long = 0,
    var teamTime: Long = 0,
)
